server/pbench/bin/pbench-backup-tarballs

```diff
@@ -64,6 +64,9 @@ if [ -z "$(find $rla -maxdepth 0 -type d -empty 2>/dev/null)" ] ;then
     fi
 fi
 
+# Backup to s3
+$BINDIR/pbench-backup-to-s3
```

Comment:
We want to re-work this commit so that `pbench-backup-tarballs` becomes a python3 executable that performs everything done by the current script, and adds the S3 backup behavior.

Reply:
WIP